packages/preview/echarm/0.1.0/typst.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "echarm"
+version = "0.1.0"
+entrypoint = "lib.typ"
+authors = ["lublak"]
+license = "MIT"
+description = "Run echarts in typst with the use of CtxJS."
+
+homepage = "https://github.com/lublak/typst-echarm-package"
+repository = "https://github.com/lublak/typst-echarm-package"
```

Comment:
specifying homepage and repository is redundant, just repository suffices

Reply:
I can change it, no problem. Just for your information. It's in a lot of packages in this (also my last one for ctxjs). I just looked in one, because I was unsure what I can use as a homepage.

Reply:
> It's in a lot of packages in this (also my last one for ctxjs).

Yeah, I noticed that in ctxjs, but didn't want to bother you for this small thing. But since I had another comment on typst.toml anyway, I figured I might as well mention it. But it's not really important.

Reply:
:) okay! In ctxjs I will remove it in the next release. (I think it makes no sense to create a new release for it).